CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # master
 *Please put new entries at the top.
 
+# 8.0.0-rc.2
```

Comment:
Sorry, still not quite right. 😞 This shouldn't have a header. We add the header when we make the release. The next release may or not be rc2; we don't want to assume.